tensorflow/compiler/mlir/tools/kernel_gen/tests/tf_64bit_indexer.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: kernel-gen-opt %s --split-input-file \
+// RUN:   --tf-to-jit-invocation="tile-sizes=1,2,3 \
+// RUN:   unroll-factors=3,2,1 max-supported-rank=32 \
+// RUN:   enable-ftz=false index_64bit=false cpu-codegen=false \
+// RUN:   jit_i64_indexed_for_large_tensors=true" | \
+// RUN: FileCheck %s
+
+// CHECK-LABEL: @unary_tanh_rint
+// CHECK-SAME: (%[[ARG:.*]]: tensor<*xf32>)
+func @unary_tanh_rint(%arg : tensor<*xf32>) -> (tensor<*xf32>) {
+  // CHECK:      %c4294967296 = arith.constant 4294967296 : index
```

Comment:
Can `c4294967296` get a name like the values below?

Reply:
named accordingly.